### PR TITLE
Techbiz_tutorial_first draft

### PR DESCRIPTION
**Pull request by meenakartik**
Hi Siva and Natasha,

Pls help to review techbiz_tutorial first draft. Thanks!

### CODE REVIEW

**Review by natashascribes — COMMENTED** [review]
Hi @meenakartik , As discussed, kindly take a look at the following suggestions:

1. Please use H1, H2, and H3 hierarchically. If you feel you need to use another H1 within doc, consider splitting the topic.
2. Please user periods to indicate a complete sentence. 
3. If using Note, Tip, Important, kindly indicate the same with word to make it clear for the user.
4. If you are using sub-bullets, consider nesting them by using tab key on laptop.
5. To indicate UI items, please use "bold" formatting rather than quotes(""). 
6. As per MSTP, we do not use apostrophe (e.g. agencies' subscription). You may reword to write agency subscription.
7. Make sure that step result is indicated as a step result and not as another step.
8. Finally, please check how the output will be rendered to help you make the suitable decision on formatting, especially headings!

**Review by natashascribes — COMMENTED** [review]
@meenakartik Please check your email for detailed comments in a word doc. I could review just one file. Thank you. 
Regards,
Natasha

**Comment by lamshiling** [comment]
Hi

**Comment by meenakartik** [comment]
Hi Denise and Shiling,

Pls help to review the techbiz tutorial draft. Thanks!

File: docs/_sidebar.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 - [Domain Terms](domain_terms.md)
 - [Usage API](usage_api.md)
 - [Onboarding Flow](onboarding_flow.md)
+- [TechBiz Tutorial](techBiz_tutorial.md)
```
Comment (SenthilKumarSivakami) [inline docs/_sidebar.md:4]: @meenakartik  pls look into this.

File: docs/_sidebar.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+- [Invite Users](invite_users.md)
```
Comment (SenthilKumarSivakami) [inline docs/_sidebar.md:5]: change it to Invite users

File: docs/invite_users.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+# Invite Users
+
+Subscription Admin or any Public Officer (using their organisation email) can trigger TechPass account creation for Public Officers / Vendors from the TechBiz Portal itself.
```
Comment (SenthilKumarSivakami) [inline docs/invite_users.md:3]: pls remove brackets.

**Pull request merged**